whisk/activation.go

```diff
@@ -216,9 +216,28 @@ func (s *ActivationService) Get(activationID string) (*Activation, *http.Respons
 		return nil, resp, err
 	}
 
+	if a.Success == false {
+		a.StatusCode = GetStatusCodeForErrorMessage(a.Status)
+	}
+
 	return a, resp, nil
 }
 
+func GetStatusCodeForErrorMessage(msg string) (int) {
```

Comment:
I suppose it's ok to have this function be public. 
You can unit test just this function.
Note that to incorporate this fix into the `wsk` CLI, we'll need to bump the godep https://github.com/apache/openwhisk-cli/blob/678deee9510abac72b84c7f3ad98a0f58b97443e/Godeps/Godeps.json#L9